docs/source/implementations/labeling_raw_return.rst

```diff
@@ -0,0 +1,79 @@
+.. _implementations-labeling_raw_return:
+
+===========
+Raw Returns
+===========
+
+Labeling data by raw returns is the most simple and basic method of labeling financial data for machine learning. Raw returns can
+be calculated either on a percentage or logarithmic basis. Using returns rather than raw prices is usually preferred for financial data
+analysis due to returns being automatically scaled with respect to the underlying. Unlike raw prices, returns are better suited for
+mathematical analysis due to the possibility of negative returns. The percentage raw return for an observation with
```

Comment:
The main benefit is that returns are thought to be stationary where prices are not. Please read more about it and then include it.

Reply:
Add to notebook.